package.json

```
@@ -20,6 +21,7 @@
     "@typescript-eslint/parser": "2.25.0",
     "babel-plugin-styled-components": "1.10.7",
     "chalk": "3.0.0",
+    "d3": "^5.15.0",
```

Can we use exact dependencies versions? Just because we're not committing the lockfile.

```suggestion
    "d3": "5.15.0",
```

(This is a decision I made when I set up this repo that I think is good, but I could be wrong. We can talk about it!)